lang/cs/Org.Apache.REEF.Driver/Org.Apache.REEF.Driver.csproj

```diff
@@ -174,6 +174,13 @@ under the License.
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
+  <Import Project="..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets" Condition="Exists('..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets')" />
```

Comment:
Many scripts in csproj were auto generated, but we do clean them up and moved the shared stuff into build.prop to make each csproj file clean, and easy for maintenance. I agree with @markusweimer to move those into build.prop file. This way would keep the change minimum. If we need to update later, we just need to update at once place. .